### PR TITLE
feat: added explore page and authentication

### PR DESCRIPTION
**Pull request by techieshubham30**

### CODE REVIEW

**Review by manojasarsa — APPROVED** [review]
The code is pretty structured and clean.
GTG!

File: src/contexts/VideosContext.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const VideosContext = createContext();
+
+const VideosProvider = ({ children }) => {
+  const [videos, setVideos] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const { data } = await axios.get("api/videos");
+        setVideos(data.videos);
```
Comment (msdeshmukh009) [inline src/contexts/VideosContext.js:13]: Nothing major but you can add a check here for status.

File: src/pages/Signin/Signin.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+        <button
+          className="login-guest-btn btn-primary-outline"
+          onClick={(e) => {
+            e.preventDefault();
+            setFormData({
+              email: "adarshbalika@gmail.com",
+              password: "adarshBalika123",
+            });
+            loginHandler();
```
Comment (msdeshmukh009) [inline src/pages/Signin/Signin.js:74]: You can extract this function

**Review by msdeshmukh009 — APPROVED** [review]
The authentication is working as expected. Good to go!!

**Pull request merged**